src/errors/src/activeUser.js

```diff
@@ -1,7 +1,7 @@
 import BaseError from './base';
 
 export default class ActiveUserError extends BaseError {
-  constructor(message = 'An active user already exists.', debug, code) {
-    super('ActiveUserError', message, debug, code);
+  constructor(message = 'An active user already exists.', debug, code, kinveyRequestId) {
```

Comment:
For a client-only error like the `ActiveUserError`, what should I (the developer) expect the value of `kinveyRequestId` to be? Is this documented?